src/components/common/button/index.tsx

```diff
@@ -1,5 +1,49 @@
-const Button = () => {
-  return <div></div>;
+import { vars } from '@/styles/variants.css';
+import { CSSProperties } from 'react';
+import * as style from './style.css';
+
+export type ButtonProps = {
+  type: 'button' | 'submit';
+  fontSize: 'small' | 'medium' | 'large';
+  fontWeight: boolean;
+  textColor: string;
+  backgroundColor: string;
+  border: boolean;
+  borderColor?: string;
+  paddingSize: 'small' | 'medium' | 'full';
+  text: string;
+  style?: CSSProperties;
+};
+
+const Button = ({
+  type = 'button',
+  fontSize = 'small',
+  fontWeight = false,
+  textColor = '#fff',
+  backgroundColor = vars.color.primary,
+  border = false,
+  borderColor = vars.color.primary,
+  paddingSize = 'small',
+  text = 'button',
+  ...props
+}: ButtonProps) => {
+  const buttonStyle = {
+    fontSize,
+    fontWeight: fontWeight ? '700' : '400',
+    color: textColor,
```

Comment:
p5;
textColor 말고 그냥 color로 해도 될 것 같습니다.
변수명 지적할게 없는데 굳이 찾아서 만든 거여서 무시하셔도 됩니다ㅎㅎ 
```suggestion
    color,
```

Reply:
생각해보니 fontSize, fontWeight인데 폰트 색상은 textColor로 해두었네요..ㅎㅎㅎ color로 하기에는 다른 color 필드와 헷갈릴 것 같아 fontColor로 변경했습니다! 좋은 지적 감사합니당